src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunItems.java

```diff
@@ -5,6 +5,9 @@
 import java.util.List;
 import java.util.Map;
 
+import io.github.thebusybiscuit.slimefun4.implementation.items.magical.staves.FireStaff;
```

Comment:
still not in right place, should put right after 
```java
import io.github.thebusybiscuit.slimefun4.core.attributes.Radioactivity;
```

Reply:
ah okay

Reply:
done